detect.go

```diff
@@ -31,7 +31,7 @@ func Detect(versionParser VersionParser) packit.DetectFunc {
 			},
 		}
 
-		_, err := os.Stat(filepath.Join(context.WorkingDir, ConfFile))
+		_, err := os.Stat(getNginxConfLocation(context.WorkingDir))
```

Comment:
If you wanted to make this slightly cleaner you could use the [`fs.Exists()` method in packit](https://github.com/paketo-buildpacks/packit/blob/2247967a3f873b178f6fb16c5e6411646ca0882a/fs/exists.go#L9).

No reason to block this PR on that change, though.

Reply:
@robdimsdale  Let's make this change later